trendmicro_apex/plugin.spec.yaml

```diff
@@ -0,0 +1,119 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: trendmicro_apex
+title: Trend Micro Apex
+description: Report suspicious objects and files to block on APEX firewall
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+enable_cache: false
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/trendmicro_apex
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.trendmicro.com/en_ca/business/products/user-protection/sps/endpoint.html
+tags:
+- trend_micro
+- incident management
+- reporting
+hub_tags:
+  use_cases: [threat_detection_and_response, reporting_and_analytics]
+  keywords: [suspicious objects, incident reporting, response, trend micro, trendmicro, apex]
+  features: []
+connection:
+  hostname:
+    title: Hostname
+    description: URL of the Apex Security Manager
+    type: string
+    required: true
+  application_id:
+    title: Application ID
+    description: Application ID to communicate to the Apex Security Manager
+    type: credential_secret_key
+    required: true
+  api_key:
+    title: API Key
+    description: API key paired with the Application ID
+    type: credential_secret_key
+    required: true
+actions:
+  add_file_to_udso_list:
+    title: Add File to UDSO List
+    description: Add a file to the UDSO list of the Apex Security Manager
+    input:
+      file:
+        title: File
+        type: file
+        description: File to be marked as suspicious
+        required: true
```

Comment:
Needs input example

Reply:
@jschipp-r7 I don't understand how I can provide an example of input type "file".  It isn't a string or integer, it would be an uploaded file with contents and a filename.

Reply:
@tylerslijboom-r7 

```suggestion
        required: true
        example: '{ "filename": "setup.exe", "content":"TVqQAAMAAAAEAAAA//8AALgAAAAAAAAAQAAAAAAAAAAAAAAA..." }
```

Reply:
done as a JSON object